components/core/src/fs.rs

```diff
@@ -258,6 +261,32 @@ pub fn find_command(command: &str) -> Option<PathBuf> {
     }
 }
 
+/// Returns the absolute path to the given command from a given package installation.
+///
+/// If the command is not found, then `None` is returned.
+///
+/// # Failures
+///
+/// * The path entries metadata cannot be loaded
+pub fn find_command_in_pkg(command: &str,
+                           pkg_install: &PackageInstall,
+                           fs_root_path: &Path)
+                           -> Result<Option<PathBuf>> {
+    for path in try!(pkg_install.paths()) {
+        let stripped = path.strip_prefix("/").expect("Package path missing / prefix");
+        let candidate = fs_root_path.join(stripped).join(command);
+        if candidate.is_file() {
+            return Ok(Some(path.join(command)));
+        } else {
+            match find_command_with_pathext(&candidate) {
```

Comment:
Since this is Windows specific behaviour, should we not include it if we're not in a Windows binary?